### PR TITLE
update suse information regarding to repositories fixes #29194 

### LINKED ISSUE #29194: Documentation - SuSE 11 SP2 repo no longer available
https://docs.saltstack.com/en/latest/topics/installation/suse.html
The repo referred to in the SuSE 11 SP2 instructions is no longer available. This should probably be removed from documentation.

### PR DESCRIPTION
**Pull request by aboe76**
Caution!!! don't merge without checking.

This requires coordination between salt, salt-bootstrap and suse/opensuse.
see https://github.com/saltstack/salt-bootstrap/issues/706

We need to make sure that everything in salt-bootstrap is also ready.
and all repo's are working perfectly.

I don't know if salt also wants a announcement of this change,
because a lot of suse / opensuse users using the old repo's

Please wait until I give the go ahead

### CODE REVIEW

**Comment by aboe76** [comment]
Talked to Suse and this is a go for merge !!

File: doc/topics/installation/suse.rst [hunk 1]
```diff
@@ -2,14 +2,19 @@
 SUSE Installation
 =================
 
-With openSUSE 13.2, Salt 2014.1.11 is available in the primary repositories.
-The devel:language:python repo will have more up to date versions of salt,
-all package development will be done there.
+Since openSUSE 13.2, Salt 2014.1.11 is available in the primary repositories.
+With the release of suse manager 3 a new repository setup has been created.
```
Comment (tampakrap) [inline doc/topics/installation/suse.rst]: SUSE Manager 3, the capitalization is important as it is branded

File: doc/topics/installation/suse.rst [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
+The new repo will by systemsmanagement:saltstack, this repo will be the source
```
Comment (tampakrap) [inline doc/topics/installation/suse.rst]: I would rephrase that to: "The new repo is systemsmanagement:saltstack, which is the source...."

File: doc/topics/installation/suse.rst [hunk 3]
```diff
@@ -2,14 +2,19 @@
...
+for newer stable packages. For backward compatibilty a linkpackage will be
```
Comment (tampakrap) [inline doc/topics/installation/suse.rst]: backwards instead of backward? (not entirely sure about this)

**Comment by tampakrap** [comment]
apart from the minor comments above, :+1: thanks for working on this

File: doc/topics/installation/suse.rst (hunk 3 above)
Comment (aboe76) [inline doc/topics/installation/suse.rst]: @tampakrap see https://en.wikipedia.org/wiki/Backward_compatibility, so I think this is correct

**Comment by cachedout** [comment]
@aboe76 Please let me know when this is ready to go. Thanks!

**Comment by aboe76** [comment]
You can merge It @cachedout

**Comment by cachedout** [comment]
Thanks, @aboe76
